examples/RFPowerSpectraAttenuation.ipynb

```diff
@@ -0,0 +1,372 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Count:\t33722\n",
+      "Min:\t0.0000\tdB/(MHz*cm)\n",
+      "Max:\t26.7213\tdB/(MHz*cm)\n",
+      "Mean:\t0.9046\tdB/(MHz*cm)\n",
+      "Sigma:\t1.3161\tdB/(MHz*cm)\n",
+      "Variance: 1.7322 dB/(MHz*cm)\n",
+      "Sum:\t30505.51 dB/(MHz*cm)\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "statistics_filter = itk.LabelStatisticsImageFilter[type(metric_image),type(output_mask_image)].New()\n",
+    "statistics_filter.SetInput(metric_image)\n",
+    "statistics_filter.SetLabelInput(output_mask_image)\n",
+    "statistics_filter.Update()\n",
+    "\n",
+    "print(f'Count:\\t{statistics_filter.GetCount(1)}\\n'\n",
+    "      f'Min:\\t{statistics_filter.GetMinimum(1):0.4f}\\tdB/(MHz*cm)\\n'\n",
+    "      f'Max:\\t{statistics_filter.GetMaximum(1):0.4f}\\tdB/(MHz*cm)\\n'\n",
+    "      f'Mean:\\t{statistics_filter.GetMean(1):0.4f}\\tdB/(MHz*cm)\\n'\n",
+    "      f'Sigma:\\t{statistics_filter.GetSigma(1):0.4f}\\tdB/(MHz*cm)\\n'\n",
+    "      f'Variance: {statistics_filter.GetVariance(1):0.4f} dB/(MHz*cm)\\n'\n",
+    "      f'Sum:\\t{statistics_filter.GetSum(1):0.2f} dB/(MHz*cm)\\n'\n",
+    "      )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Histogram size: 100000 bins from 0.0 dB/(MHz*cm) to 26.7213 dB/(MHz*cm) with 33722 entries\n",
+      "Histogram mean is 0.9047\n",
+      "9295 entries in bin  from 0.0 dB/(MHz*cm) to 0.0003 dB/(MHz*cm)\n",
+      "50th quantile: 0.5667 dB/(MHz*cm)\n",
+      "75th quantile: 1.2394 dB/(MHz*cm)\n",
+      "95th quantile: 2.9658 dB/(MHz*cm)\n"
+     ]
+    }
+   ],
+   "source": [
+    "histogram_filter = itk.MaskedImageToHistogramFilter[type(metric_image),type(output_mask_image)].New()\n",
+    "histogram_filter.SetInput(metric_image)\n",
+    "histogram_filter.SetMaskImage(output_mask_image)\n",
+    "histogram_filter.SetMaskValue(1)\n",
+    "histogram_filter.SetHistogramSize([1e5])\n",
+    "histogram_filter.SetMarginalScale(10)\n",
+    "histogram_filter.Update()\n",
+    "\n",
+    "histogram = histogram_filter.GetOutput()\n",
+    "size = histogram.GetSize(0)\n",
+    "print(f'Histogram size: {size} bins'\n",
+    "      f' from {histogram.GetBinMin(0,0)} dB/(MHz*cm)'\n",
+    "      f' to {histogram.GetBinMax(0,size-1):0.4f} dB/(MHz*cm)'\n",
+    "      f' with {histogram.GetTotalFrequency()} entries')\n",
+    "\n",
+    "print(f'Histogram mean is {histogram.Mean(0):0.4f}')\n",
+    "\n",
+    "print(f'{histogram.GetFrequency(0, 0)} entries in bin '\n",
+    "      f' from {histogram.GetBinMin(0,0)} dB/(MHz*cm)'\n",
+    "      f' to {histogram.GetBinMax(0,0):0.4f} dB/(MHz*cm)')\n",
+    "print(f'50th quantile: {histogram.Quantile(0,0.5):0.4f} dB/(MHz*cm)')\n",
+    "print(f'75th quantile: {histogram.Quantile(0,0.75):0.4f} dB/(MHz*cm)')\n",
```

Comment:
Also, 20/40/60/80 are called quantiles, 25/50/75 are called quartiles (I think) and arbitrary numbers such as 5% are called percentiles. Let's call them all percentiles in this output.

Reply:
Good catch, fixed